crates/nanocld_client/src/process.rs

```diff
@@ -213,6 +213,18 @@ impl NanocldClient {
       .await?;
     Ok(Self::res_stream(res).await)
   }
+  /// The stats are streamed as a [Receiver](Receiver) of [stats](Stats)
+  /// It get the stats by key name(returned by nanocl ps) instead of name and kind
+  pub async fn stats_processes_by_name(
```

Comment:
Should be `stats_process_by_name` also add an example pls

Reply:
Also add a test pls at the bottom of the file

Reply:
Problem is the example aren't async, which make you write more code.
We can start to do it this way and run the test with doc tests but it have to be change everywhere then so.. We can do that but in a dedicated PR

Reply:
Added the example